src/test/store/symbolication.test.js

```diff
@@ -202,6 +273,126 @@ describe('doSymbolicateProfile', function() {
       );
     });
   });
+
+  it('can symbolicate a profile with a partial symbol table and re-symbolicate it with a complete symbol table', async () => {
+    // Get rid of any cached symbol tables from the previous tests.
+    await _deleteDatabase(`${symbolStoreName}-symbol-tables`);
```

Comment:
Optional (since it's not introduced by your patch): This seems like it should be a `beforeEach()`, and not done manually in each test. Conceptually each `it` block should be as pure as possible to help in reproducibility.

Reply:
I briefly attempted this but then ran into timeouts of the deleteDatabase call that I couldn't explain, so I'm abandoning it.